src/main/java/nextstep/subway/domain/ShortestDistancePathFinder.java

```diff
@@ -0,0 +1,63 @@
+package nextstep.subway.domain;
+
+import nextstep.subway.application.dto.PathResponse;
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.WeightedMultigraph;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@Service
+@Transactional(readOnly = true)
```

Comment:
도메인 패키지 내부에 있어서 Service라는 용어를 조금 어색하다고 느껴지는데요
Service 대신 Component 애노테이션을 사용하면 어떨까요?
```suggestion
@Component
```